src/lib/loaders/loaders_without_authentication/gravity.ts

```diff
@@ -202,7 +202,7 @@ export default opts => {
       { headers: true }
     ),
     setLoader: gravityLoader(id => `set/${id}`),
-    setsLoader: gravityLoader("sets"),
+    setsLoader: gravityLoader("sets", {}, { headers: true }),
```

Comment:
Just wondering why this wouldn't be the default?

Reply:
This actually changes the return type of the data. Instead of getting the data back directly from the Promise, you'd get back `{body:. headers:}` and thus need to destructure `body` instead of resolving directly in any places that invoke this loader.